testsuite/unit-tests/src/test/java/org/jboss/resteasy/test/util/CaseInsentiveMapTest.java

```diff
@@ -12,6 +13,17 @@
  */
 public class CaseInsentiveMapTest {
 
+   protected CaseInsensitiveMap<String> map;
```

Comment:
There's an existing method `testMap` below that's creating a new instance of `map`. Can that method use this map instead for consistency? (wasn't able to add my comment on that line).

Reply:
Great point.  I actually think that was the intent, and we just missed it.